solutions/pagination-strategy/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "vercel-example--pagination-strategy",
```

Comment:
Is this an issue with plop? I was expecting the name to be `pagination-strategy`

Reply:
You know. Now I think it's plop issue. When I created plop I thought prefix would be better but I think you're right. it should be just a name without any prefix.

Reply:
https://github.com/vercel/examples/pull/150